ydb/core/persqueue/pq_impl.cpp

```diff
@@ -4424,6 +4431,30 @@ void TPersQueue::Handle(TEvPQ::TEvPartitionScaleStatusChanged::TPtr& ev, const T
     }
 }
 
+void TPersQueue::Handle(TEvPQ::TEvDeletePartitionDone::TPtr& ev, const TActorContext& ctx)
+{
+    auto* event = ev->Get();
+    const ui64 writeId = event->Cookie;
```

Comment:
Return writeId in EvDeletePartitionDone response, no Cookie field required anymore.

Reply:
`PartitionId` is used instead of `WriteId`
https://github.com/ydb-platform/ydb/pull/4407/commits/76c2d74fad557fb217fc4d91fc813ada28978cf8